guacamole/src/main/webapp/app/client/templates/client.html

```diff
@@ -151,7 +151,7 @@ <h3>{{'CLIENT.SECTION_HEADER_DISPLAY' | translate}}</h3>
                     <div class="content">
                         <div id="zoom-settings">
                             <div ng-click="zoomOut()" id="zoom-out"><img src="images/settings/zoom-out.png" alt="-"/></div>
-                            <div id="zoom-state">{{formattedScale()}}%</div>
```

Comment:
If this is no longer necessary, then we should probably also delete:

https://github.com/apache/guacamole-client/blob/91f19b2e03cce32c964f1f06e810d84aa8b924d5/guacamole/src/main/webapp/app/client/controllers/clientController.js#L734-L736

Reply:
Yep, removed.